conditionals.py

```diff
@@ -210,4 +210,22 @@ def exclude_mapcompass_info_remove(random_settings, weight_dict, **kwargs):
 def ohko_starts_with_nayrus(random_settings, weight_dict, extra_starting_items, **kwargs):
     """ If one hit ko is enabled, add Nayru's Love to the starting items """
     if random_settings['damage_multiplier'] == 'ohko':
-        extra_starting_items['starting_items'] += ['nayrus_love']
\ No newline at end of file
+        extra_starting_items['starting_items'] += ['nayrus_love']
+
+def invert_dungeons_mq_count(random_settings, weight_dict, **kwargs):
+    """ When activated will invert the MQ dungeons count
+        kwargs: [chance of having the MQ count reversed]
+    """
+    if random_settings['mq_dungeons_mode'] != 'count':
+        return
+
+    chance_of_inverting_mq_count = int(kwargs['cparams'][0])
+    invert_mq_count = random.choices([True, False], weights=[chance_of_inverting_mq_count, 100-chance_of_inverting_mq_count])[0]
+
+    if not invert_mq_count:
+        return
+    
+    current_mq_dungeons_count = int(random_settings['mq_dungeons_count'])
```

Comment:
Can I get you to just do this inline in the next line rather than making a new variable?

Reply:
I like having python following "explicit over implicit" rule and with `new_mq_dungeons_count = 12 - int(random_settings['mq_dungeons_count'])` I'm not sure the code will be as much understandable as it is now ?

I was also hesitating having a variable `total_dungeons_count = 12` so I would have something like `new_mq_dungeons_count = total_dungeons_count - current_mq_dungeons_count` and a bit much clearer to read.

Reply:
@matthewkirby any advice on it ? If you still want me to inline this variable there's no problem, I've just wanted to give you my explanation on why I did it that way :)

Reply:
This is totally fine. Thanks for the explanation!